sklearn/svm/classes.py

```diff
@@ -582,6 +586,17 @@ class SVC(BaseSVC):
         more information on the multiclass case and training procedure see
         section 8 of [1]_.
 
+    classes_ : array, shape = [n_class]
+        Sorted unique classes as provided in the target vector ``y`` in the
+        ``fit`` method.
+
+    class_weight_: array, shape = [n_class]
```

Comment:
missing space after underscore before colon: `class_weight_ : array, shape = [n_class]`

Reply:
Thanks for pointing out